gdx/src/com/badlogic/gdx/graphics/g3d/utils/FirstPersonCameraController.java

```diff
@@ -26,17 +26,20 @@
 /** Takes a {@link Camera} instance and controls it via w,a,s,d and mouse panning.
  * @author badlogic */
 public class FirstPersonCameraController extends InputAdapter {

```

Comment:
API should be documented in CHANGES file, something like "FirstPersonCameraController key mapping is now configurable"

Reply:
done